qa/standalone/erasure-code/test-erasure-eio.sh

```diff
@@ -281,10 +289,10 @@ function TEST_rados_get_subread_eio_shard_0() {
 
 function TEST_rados_get_subread_eio_shard_1() {
     local dir=$1
-    setup_osds || return 1
+    setup_osds 4 || return 1
```

Comment:
3 OSDs would suffice for an erasure pool of "2+1" .

Reply:
I'm taking 1 OSD down/out which can begin recovery to the 4th OSD.  Not sure this is critical to this test, but I'd rather be using a more realistic scenario.

Reply:
oh, i see.